waku/waku.md

```diff
@@ -270,6 +272,48 @@ This feature will likely stop being experimental in v1.
 
 It is currently bounded to a maximum of 1000 topics. If you are interested in more topics than that, this is currently underspecified and likely requires updating it. The constant is subject to change.
 
+**Message Confirmations**
+
+Message confirmations tell a node that a message originating from it has been received by its peers, allowing a node to know whether a message has or has not been received.
+
+A node MAY send a message confirmation for any batch of messages received with a packet Messages Code (`0x01`).
+
+A message confirmation is sent using Batch Acknowledge packet (`0x0b`) or Message Response packet (`0x0c`).
+
+The Batch Acknowledge packet is followed by a keccak256 hash of the envelopes batch data (raw bytes).
+
+The Message Response packet is more complex and is followed by a Versioned Message Response. The current version of the message response MUST be equal to `1`.
```

Comment:
Do old clients ignore newer versions?

@adambabik

Reply:
Yes, the version is checked and if it's different then the message is ignored. This is the only place were there is a versioned payload.